cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -145,5 +146,32 @@
                 'state':    'DC',
                 'zip_code': '20552-0003'
         }
-    }) }}
+        }) }}
+    {% endif %}
+
+    {% for block in page.sidefoot %}
+        <div class="block block__flush-top">
+            {% if 'related_posts' in block.block_type %}
+                {{ related_posts.render(block) }}
+            {% else %}
+                {% if 'heading' in block.block_type %}
+                    <h3 class="o-sidebar-breakout_heading">
+                        {{ render_stream_child(block) }}
+                    </h3>
+                {% elif 'slug' in block.block_type %}
+                    <h2 class="header-slug">
+                        <span class="header-slug_inner">
+                            {{ render_stream_child(block) }}
+                        </span>
+                    </h2>
+                {% elif 'paragraph' in block.block_type %}
+                    <div class="short-desc">
```

Comment:
@jimmynotjim nah remember wagtail ignores those tags. non-formatted text in richtext fields like this by default are surrounded by `p` tags. so having a div with the assigned class name creates the same effect.